pom.xml

```
@@ -6,7 +6,7 @@
 
     <groupId>org.java</groupId>
     <artifactId>icing</artifactId>
-    <version>2020.09.21</version>
+    <version>2020.10.25</version>
```

```suggestion
    <version>2020.10.30</version>
```

The date we'll merge it once everything's complete.